packages/terra-site/src/examples/modal/DefaultWithDialog.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Modal from 'terra-modal';
+import Dialog from 'terra-dialog';
+
+class ModalIsOpen extends React.Component {
+  constructor() {
```

Comment:
Do we need to pass props into the constructor? It looks like our other react components are doing that